src/shared/components/lollipopMutationPlot/styles.scss

```diff
@@ -0,0 +1,29 @@
+.fade-out {
+  -webkit-transition: opacity 0.6s, visibility 0s 0.6s;;
+  -moz-transition: opacity 0.6s, visibility 0s 0.6s;;
+  -o-transition: opacity 0.6s, visibility 0s 0.6s;;
+  transition: opacity 0.6s, visibility 0s 0.6s;;
+  opacity: 0;
+  visibility:hidden;
+}
+.fade-in {
+  -webkit-transition: opacity 0.2s;
+  -moz-transition: opacity 0.2s;
+  -o-transition: opacity 0.2s;
+  transition: opacity 0.2s;
+  opacity: 1;
+}
+.collapse-out {
```

Comment:
Oh, that one. I think we can live without the animation. And if we really need the animation, there is this library `react-collapse`, which is already included in the project.